opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -104,6 +104,15 @@ class Well {
 
     using GasInflowEquation = WellGasInflowEquation;
 
+    enum WellRefinementType {
+        STANDARD,
+        LGR,
+        MIXED,
+    };
```

Comment:
Are users, or more specifically is user code, expected to know/care about the `WellRefinementType`?  If not, I think this `enum` should be a `private:` implementation detail.  Moreover, it should **probably** be a "strong" enum&ndash;i.e., an `enum class`&ndash;instead of "bare" enum as well.